course5/homework03/homework03.cc

```diff
@@ -1,6 +1,7 @@
 #include <cstdio>
-
-// TODO your code goes here
+#include <iostream>
+#include <string>
+#include "print.h"
```

Comment:
It's better to order headers from the most local to the most general. So the std headers should always go at the end.